From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 11 Oct 2011 09:30:22 -0600
Patch-mainline: v3.2-rc1
Subject: [PATCH] hv: remove unused LOWORD and HIWORD macros from hyperv.h
Git-commit: 5557e8a60570d0c2f3a06d6e9e6a0f5074c313f2

They aren't used anywhere anymore now that the debugging macros are
gone, so remove it from hyperv.h as well.

Cc: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 include/linux/hyperv.h |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/include/linux/hyperv.h b/include/linux/hyperv.h
index 7211e2c..6d9a53f 100644
--- a/include/linux/hyperv.h
+++ b/include/linux/hyperv.h
@@ -694,10 +694,6 @@ extern void vmbus_get_debug_info(struct vmbus_channel *channel,
 extern void vmbus_ontimer(unsigned long data);
 
 
-#define LOWORD(dw) ((unsigned short)(dw))
-#define HIWORD(dw) ((unsigned short)(((unsigned int) (dw) >> 16) & 0xFFFF))
-
-
 #define VMBUS				0x0001
 #define STORVSC				0x0002
 #define NETVSC				0x0004
-- 
1.6.0.2


